src/vs/editor/contrib/dropOrPasteInto/browser/defaultProviders.ts

```diff
@@ -200,6 +200,19 @@ class PasteHtmlProvider implements DocumentPasteEditProvider {
 	}
 }
 
+export class PasteImageProvider implements DocumentPasteEditProvider {
+
+	public readonly kind = new HierarchicalKind('image');
+
+	public readonly pasteMimeTypes = ['Files']; // this should be `image/png` etc, but how mime types are checked in copyPasteController doesn't encompass this
```

Comment:
You should be able to use `image/*`

Reply:
i think i tried that and the reason I went for `Files` was because that's the type it comes back as, which is kind of puzzling (it's a file, with the type of `image/png`, but the type is just `File`)

![Screenshot 2024-09-23 at 1 25 16 PM](https://github.com/user-attachments/assets/b064ea88-002b-4ab3-8e09-f9a4941a323b)